lib/matplotlib/backends/backend_svg.py

```diff
@@ -26,196 +26,120 @@
 from matplotlib.transforms import Affine2D, Affine2DBase
 from matplotlib import _png
 
-from xml.sax.saxutils import escape as escape_xml_text
+from xml.etree import ElementTree as etree
 
 backend_version = __version__
 
-# ----------------------------------------------------------------------
-# SimpleXMLWriter class
```

Comment:
@OceanWolf I may have not looked into this as much as I should have.  I wanted to keep the API of the old XMLWriter class (at least to the extent that it is used within MPL).  I believe the simplest method to do this is by extending the xml.etree.TreeBuilder class.  I also wanted to be sure that the output XML was formatted the same so that all the tests of XML results all passed, so the XMLWriter class was still necessary.